app/controllers/meetings_controller.rb

```
@@ -0,0 +1,49 @@
+class MeetingsController < ApplicationController
+  def index
+    load_meetings
+  end
+
+  def new
+    fraternity
+    @meetings_form = MeetingForm.new
+  end
+
+  def create
+    meeting_form = MeetingForm.new(meetings_params)
+
+    if meeting_form.valid?
+      GenerateMeetings.call(fraternity: fraternity, organizers: fraternity.users, **meeting_form.attributes)
+      
+      redirect_to fraternity_meetings_path(fraternity), notice: notification_for(:created, Meeting)
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def meetings_params
+    params.fetch(:meeting, {}).permit(:starts_on, :ends_on, :frequency, :organizers)
+  end
+
+  def build_meeting
+    @meeting ||= meetings_scope.build
+    @meeting.attributes = meetings_params
+  end
+
+  def load_meeting
+    @meeting ||= meetings_scope.find_by(id: params[:id])
```

This should be `find` instead of `find_by`. We want to throw an exception if the record doesn't exist.